packages/components-library/src/components/filters/MultiFilter.vue

```diff
@@ -0,0 +1,272 @@
+        :name="name"
+        :placeholder="placeholder"
+        trim
+      />
+      <b-input-group-append>
+        <b-button
+          variant="outline-secondary"
+          :disabled="isLoading"
+          @click.prevent="query= ''"
+        >
+          <font-awesome-icon
+            v-if="isLoading"
+            icon="spinner"
+            class="fa-spin"
+            size="xs"
+          />
+          <font-awesome-icon
+            v-else
+            icon="times"
+          />
+        </b-button>
+      </b-input-group-append>
+    </b-input-group>
+    <b-form-checkbox-group
+      v-if="slicedOptions.length"
+      v-model="selection"
+      class="checkbox-list"
+      :name="name"
+      :options="slicedOptions"
+      stacked
+    />
+
+    <b-link
+      v-if="showCount < multifilterOptions.length"
+      class="card-link"
+      @click="showMore"
+    >
+      {{ showMoreText }}
+    </b-link>
+    <font-awesome-icon
+      v-if="foundOptionCount >= 100"
+      v-b-popover.hover="'There are 100 or more results found, only the first 100 are available. Please refine your search.'"
+      icon="exclamation-triangle"
+      class="warning text-danger"
+    />
+  </div>
+</template>
+
+<script>
+import { library } from '@fortawesome/fontawesome-svg-core'
+import {
+  faSpinner,
+  faTimes,
+  faExclamationTriangle
+} from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+library.add(faTimes, faExclamationTriangle, faSpinner)
+
+export default {
+  components: { FontAwesomeIcon },
+  props: {
+    /**
+     * The HTML input element name.
+     */
+    name: {
+      type: String,
+      required: true
+    },
+    /**
+     * The HTML input element placeholder.
+     */
+    placeholder: {
+      type: String,
+      required: false,
+      default: () => 'Type to search more'
+    },
+    /**
+     * The async method returning filter options.
+     */
+    options: {
+      type: Function,
+      required: true
+    },
+    /**
+     * The amount of initial options to show.
+     */
+    initialDisplayItems: {
+      type: Number,
+      required: false,
+      default: () => 1
+    },
+    /**
+     * @model
+     */
+    value: {
+      type: Array,
+      default: () => []
+    },
+    /**
+     * The amount of options to show after filtering the checkbox options.
+     */
+    maxVisibleOptions: {
+      type: Number,
+      default: () => 10
+    }
+  },
+  data () {
+    return {
+      showCount: 0,
+      isLoading: false,
+      triggerQuery: Number,
+      inputOptions: [],
+      initialOptions: [],
+      query: ''
+    }
+  },
+  computed: {
+    selection: {
+      get () {
+        return this.value
+      },
+      set (value) {
+        this.$emit('input', value.length === 0 ? undefined : value)
+      }
+    },
+    multifilterOptions () {
+      if (this.inputOptions.length > 0) return this.inputOptions
+      else return this.initialOptions
```

Comment:
Dit zorgt ervoor dat de filtering anders werkt dan je zou verwachten

Reply:
zou dat niet moeten zijn
 if ( this.inputOptions.length == 0 && queryString == leeg) dan initialOptions anders inputOptions

Reply:
fix in https://github.com/molgenis/molgenis-frontend/pull/388/commits/6640ca0d528f52d4358a8aae26e94885c75b1504